### PR TITLE
Document 1.21.2 release changes

### PR DESCRIPTION
**Pull request by keirthana**
Pending component version changes and addition of release notes link

### CODE REVIEW

File: reference/roadmap.md [hunk 1]
```diff
@@ -12,5 +12,6 @@ Anbox Cloud follows a defined release cycle with frequent minor and patch releas
 
 | Release | Target Date | What to expect| 
 |---------|-------------|---------------|
-| 1.21.2 | April 18 2024 | * Android security updates for April 2024<br/>* Bug fixes |
 | 1.22.0 | May 15 2024 | * Support for custom Android system images<br/> * Support for 64 bit only ARM/x86 systems<br/>* Stability improvements for AAOS images and VHAL<br/>* VirGL enabled by default for NVIDIA GPUs<br/>* Update to NVIDIA 550 driver series |
```
Comment (ajanon) [inline reference/roadmap.md:15]: Unrelated to the changes, but shouldn't this also have the May security updates?

File: reference/component-versions.md [hunk 2]
```diff
@@ -2,6 +2,68 @@ This documents the versions of the different components for each Anbox Cloud rel
...
+| `anbox-cloud-tests` | `1.21/stable` | 1.21.2 |
+
+### Anbox images
+
+| Name | Version |
+|----------|--------------|
+| `jammy:android13:amd64` | 1.21.2 |
+| `jammy:android13:arm64` | 1.21.2 |
+| `jammy:android12:amd64` | 1.21.2 |
+| `jammy:android12:arm64` | 1.21.2 |
```
Comment (ajanon) [inline reference/component-versions.md:64]: I'm sorry for not noticing in previous iterations, but shouldn't we also have the AAOS images there?
These would be:
```suggestion
| `jammy:android13:amd64` | 1.21.2 |
| `jammy:android13:arm64` | 1.21.2 |
| `jammy:android12:amd64` | 1.21.2 |
| `jammy:android12:arm64` | 1.21.2 |
| `jammy:aaos13:amd64` | 1.21.2 |
| `jammy:aaos13:arm64` | 1.21.2 |
```

**Review by ajanon — CHANGES_REQUESTED** [review]

File: reference/roadmap.md (hunk 1 above)
Comment (keirthana) [reply reference/roadmap.md]: Thanks for catching this, I have included it

File: reference/component-versions.md (hunk 2 above)
Comment (keirthana) [reply reference/component-versions.md:64]: Oh yes, I have included it for previous releases as well (dating back to 1.21.0). But I feel it is better to distinguish with an indicator to show that these are still developing. I have used *Experimental* for now but wondering if something else like *Beta* would be more professional.

Comment (keirthana) [reply reference/component-versions.md:64]: Decided to use experimental based on discussion in SU

**Review by jat-canonical — APPROVED** [review]
LGTM

File: reference/component-versions.md [hunk 3]
```diff
@@ -2,6 +2,69 @@ This documents the versions of the different components for each Anbox Cloud rel
...
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud` | `1.21/stable` | 182 |
+| `anbox-cloud-core` | `1.21/stable` | 189 |
+
+### Snaps
+
+| Name |  Channel | Version |
+|----------|--------------|---------|
+| `ams`    | `1.21/stable` | 1.21.2 |
```
Comment (adglkh) [inline reference/component-versions.md:48]: As documented in the Release log, for 1.21.2 release, there is no code change for charms or snaps other than anbox-cloud-appliance, hence there is no 1.21.2 version for
- ams
- aar
- ams-node-controller
- anbox-cloud-dashboard
- anbox-stream-agent
- anbox-stream-gateway
- anbox-cloud-tests

**Review by adglkh — CHANGES_REQUESTED** [review]

**Review by ajanon — APPROVED** [review]
LGTM

File: reference/component-versions.md (hunk 3 above)
Comment (keirthana) [reply reference/component-versions.md]: Updated as per discussion, Thanks

**Review by adglkh — APPROVED** [review]
LGTM

**Pull request merged**